.github/workflows/dev-image-update.yaml

```
@@ -98,19 +98,10 @@ jobs:
       gcr_tag: ${{ needs.update_image.outputs.ui_image_tag }}
       source_gcr_url: 'gcr.io/broad-jade-dev/jade-data-repo-ui'
       target_gcr_url: 'gcr.io/datarepo-public-gcr/jade-data-repo-ui'
-  action_notify:
-    runs-on: ubuntu-latest
-    if: always()
-    needs:
-      - update_image
-      - helm_tag_bump
-      - cherry_pick_image_to_production_gcr
-    steps:
-    - name: Slack job status
-      uses: broadinstitute/action-slack@v3.15.0
-      with:
-        status: ${{ contains(needs.*.result, 'failure') && 'failure' || 'success' }}
-        author_name: Integration Test
-      env:
-        GITHUB_TOKEN: ${{ secrets.BROADBOT_TOKEN }}
-        SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+  notify-slack:
+    needs: [ update_image, helm_tag_bump, cherry_pick_image_to_production_gcr ]
```

Do you need to need all the jobs here? If you need `cherry_pick_image_to_production_gcr`, it transitively depends on the other things.
```suggestion
    needs: [ cherry_pick_image_to_production_gcr ]
```

---

It seems like you wouldn't, but the examples in the documentation indicate that you should still list them all: 
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-requiring-successful-dependent-jobs